disassembler/disassembler.py

```diff
@@ -0,0 +1,9 @@
+class Disassembler:
+    def configure(self, options):
+        raise NotImplementedError("configure")
+
+    def check_version(self, skip_version_check, splat_version):
+        raise NotImplementedError("check_version")
+
+    def known_types(self):
+        raise NotImplementedError("known_types")
```

Comment:
This looks like a perfect case for using https://docs.python.org/3/library/abc.html#abc.ABC
The methods should be `@abstractmethod` too